SwiftLeeds/Views/Local/BottomSheetView.swift

```diff
@@ -34,19 +43,16 @@ struct BottomSheetView: View {
                                   fontStyle: .title2.weight(.semibold),
                                   foregroundColor: .primary)
                     ScrollView {
-                        LocalCell(label: "Food",
-                                  imageName: "takeoutbag.and.cup.and.straw.fill",
-                                  labelFontStyle: .body)
-                        LocalCell(label: "Coffee",
-                                  imageName: "cup.and.saucer.fill",
-                                  labelFontStyle: .body)
-                        LocalCell(label: "Drink",
-                                  imageName: "wineglass.fill" ,
-                                  labelFontStyle: .body)
-                        LocalCell(label: "Best of Leeds",
-                                  imageName: "mappin",
-                                  labelFontStyle: .body)
+                        ForEach(categories, id: \.self) { category in
+                            LocalCell(
+                                label: category.name,
+                                imageName: category.symbolName,
+                                labelFontStyle: .body
+                            )
+                        }
                     }
+                    .padding(.bottom, Constant.tabBarPadding) // Account for the tab bar
+                    .transition(.opacity)
```

Comment:
I've added a transition to handle just in-case someone else animates higher up the hierarchy, but I haven't explicitly added an animation in this file